### ISSUE TITLE
Feature request: Markdown Processing for Textarea field

### ISSUE THREAD

**Issue by viocassel** [Octopoos/SEBLOD#329]
Hi Sebloders!

We are more and more use markdown syntax in GitHub, Blogs, Forums, Comments, etc...

What do you think about Markdown parsers for the **Textarea field** in **Processing options**?
"Convert Markdown to HTML" ([Commonmark](https://github.com/thephpleague/commonmark/) + [Commonmark Extras](https://github.com/thephpleague/commonmark-extras/))
and
"Convert HTML to Markdown" ([Html-to-Markdown](https://github.com/thephpleague/html-to-markdown/)).

With these options the user will be able to create your own markdown editor on their SEBLOD Forms (_for example, with [Code Pack](https://www.seblod.com/store/extensions/1854), etc._) just adding  [Commonmark.js](https://github.com/jgm/commonmark.js/) or [Marked](https://github.com/chjj/marked/) or [Markdown-it](https://github.com/markdown-it/markdown-it), etc 

See example [HTML editor in UIkit](http://getuikit.com/docs/htmleditor.html).

Also, users can use ready-made editors such as [Simplemde](https://simplemde.com/), [Woofmark](https://bevacqua.github.io/woofmark/), [Markdownify](http://tibastral.github.io/markdownify/), etc.

Regards,
Vio

**Comment by addept** [comment]
Great idea!

**Comment by Ilya-Zhulin** [comment]
Good idea, now we have to made separate field to use markdown - it isn't comfortable. of course. But don't foregt about transformation `/r/n` to `<br>` or `<p>` option. It may be conflict...

**Comment by kkaazzoo** [comment]
Good thing.

**Comment by Sulpher** [comment]
That will be a good addition to existing features.

**Comment by joomla-ua** [comment]
Nice!👍

**Comment by b2z** [comment]
:+1: on it
